### ISSUE TITLE
ocperf.py event naming doesn't correspond to perf's one

### ISSUE THREAD

**Issue by RainM** [andikleen/pmu-tools#63]
Example:

$ ocperf.py record --event offcore_response.all_reads.l3_hit.hitm_other_core sleep 1
$ perf evlist                                                                                                                     
offcore_response_all_reads_l3_hit_hitm_other_core

So, ocperf.py event name contains dots, but perf's event name contains only underscore.
It confuses tools which uses perf and doesn't let to use ocperf.py as perf's wrapper.

I believe there are few solutions: 
1. rename all ocperf.py events
2. Let users to specify perf-styled event names (with underscores)

**Comment by andikleen** [comment]
With this change, will events specified with dot still work? It doesn't look like it because you replace the name mapping. Should be fine to allow specifying events with underscores, but doted events still need to work. So it would need two entries in the events hash for each event: one with dots, and another with underscores.

BTW not clear what problem this solves? You want to pass events from perf evlist back to ocperf?

**Comment by andikleen** [comment]
Ok it looks like your second patch does just that. Added thanks.

**Comment by RainM** [comment]
Events with dots will work properly. In my pull-request I added new dictionary for perf-style named events ('perf_events' field) and added one more 'elif' statement to 'getevent' method:

I try to use ocperf.py with JMH (java microbenchmarking framework). JMH is able to run perf (or any other executable if I overwrite environment variable) and parser it's result. JMH memorizes event names which I specified in parameters (I should specify names with dots because these names will be passed to ocperf.py). During perf/ocperf.py output parsing JMH become a bit confused since there is not parameter names I specified (because underlying perf outputs names with underscored only). 
Anyway, I believe it's a bit confusing: ocperf.py outputs (right, perf outputs) event names which differ from event names I passed as argument.

BTW, may be it's better to remove '.replace(".", "_")' from Event.output_newstyle method?
